### DISCUSSION TITLE
How to set the NUM_CLASSES, if I want to train only one class?

### DISCUSSION THREAD

**Discussion by AstrayChao** [facebookresearch/detectron2#2708] [Q&A]
hi, I want to train a person detector, and the NUM_CLASSES should be 1 or 2 ('person' + 'background')? 
thank you

**Comment by ppwwyyxx** [comment]
see https://detectron2.readthedocs.io/en/latest/modules/config.html#config-references

**Reply by AstrayChao** [reply]
thanks
